geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/StatParser.scala

```diff
@@ -66,8 +67,24 @@ private class StatParser extends BasicParser {
     oneOrMore(singleStat, ";") ~~> { s => if (s.length == 1) s.head else new SeqStat(s) } ~ EOI
   }
 
+  private def groupBy: Rule1[Stat] = rule {
+    "GroupBy(" ~ string ~ "," ~ oneOrMore(statString, ";") ~ ")" ~~> { (attribute, groupedStatsList) =>
```

Comment:
In theory it should work but I was't having any luck with it. What I did seems dirty but it's the only way I could get it to not greedily eat the closing ``)``. I also had to change add new string definitions that allow ``(``, ``)``, ``,``, ``;`` and ``'``.